code/Modules/Props/Types/Prop.cs

```diff
@@ -42,7 +45,8 @@ public virtual void OnInteract(MainPlayer player)
         public override void Spawn()
         {
             base.Spawn();
-            
+
+            SetupPhysicsFromModel(PhysicsMotionType.Static);
```

Comment:
thought: Maybe i really want more comments because i don't get half of the code 😂